content/docs/command-reference/remote/add.md

```diff
@@ -104,8 +104,18 @@ By default, DVC authenticates using your AWS CLI
 (if set). This uses the default AWS credentials file. To use a custom
 authentication method, use the parameters described in `dvc remote modify`.
 
-We use the `boto3` library to communicate with AWS. The following API methods
-are performed:
+S3 remotes can also be configured entirely via environment variables, e.g.:
```

Comment:
This should not be part of this PR?

I also don't understand this change ... we don't include in `dvc remote add` any specific details ... it should be part of the `dvc remote modify`

Reply:
It was out of scope based on the PR title, sorry. The scope expanded a bit during the review process.

But we do mention env vars in the Azure (and OSS) sections of `remote add`, and those are the only other types that you can config with env vars I think, at least based on `remote modify` info (maybe also GDrive but those details are in it's own guide).

Made all the env var mentions consistent in b2372e7e for now (see #2441)

Reply:
A think a lot of remotes can be configured with env vars.

All this additional config should be part of modify to my mind. Things like

```
export ..
export
dvc remote add
```

do not make much sense overall. 

```
export ...
export ...
```

are important when you run `dvc push`/`dvc pull`. It's a way to provide values to auth flow used in DVC when it tries to access. Other ways - default config files (`.aws/credentials`) or DVC config (the one you modify with `dvc remote modify`)